app/Services/Manager.php

```diff
@@ -608,4 +608,24 @@ public function sendEmail($user, $contest, $params)
             return null;
         }
     }
+
+    /**
+     * Finds if user exists in a competition for a given contest
+     *
+     * @param app\Models\Contest         $contest
+     * @param string                     $id
+     * @return array|null
+     */
+    public function findUserInContest($contest, $id)
+    {
+        $userCompetitions = $contest->competitions->filter(function ($competition, $key) use ($id) {
+            $users = $competition->users->where('northstar_id', $id);
+
+            if (! $users->isEmpty()) {
+                return $competition;
+            }
+        });
+
+        return $userCompetitions->isEmpty() ? null : $userCompetitions;
```

Comment:
Kinda same thing w/ `$userCompetitions`. Should only be one, but happens to be in collection. Might be easier to read if more generic name? thoughts?